packages/shared/src/hooks/usePersonaContext.ts

```diff
@@ -25,7 +25,7 @@ function usePersonaInformation(
     queryOwnedPersonaInformation?: (initializedOnly: boolean) => Promise<PersonaInformation[]>,
 ) {
     const { data: personas = EMPTY_LIST, refetch } = useQuery({
-        queryKey: ['@@queryOwnedPersonaInformation(false)'],
+        queryKey: ['my-own-persona-info'],
```

Comment:
we cache this because we need to fix the loading speed. your fix is wrong, you should change all other references to this key to `@@queryOwnedPersonaInformation(false)`